evaluation.py

```diff
@@ -73,7 +104,189 @@ def draw_efficeincy(reco_Pt: np.array, cnn_l1_pt: np.array, bp_l1_pt: np.array)
     legend.AddEntry(eff_bp, "Bit Pattern ", "l")
     legend.Draw()
     cnvs.Draw()
-    cnvs.SaveAs(f"results/efficiency.png")
+    cnvs.SaveAs(f"results/histograms/efficiency.png")
```

Comment:
I agree there should be a new directory in the results with just histograms.

Please change the `README` instructions to create this folder, i.e. `mkdir -p results/histograms`